packages/nvidia_nat_core/src/nat/eval/dataset_store/register.py

```
@@ -0,0 +1,67 @@
+
+@register_dataset_store(config_type=EvalDatasetXlsConfig)
+async def register_xls_dataset_store(config: EvalDatasetXlsConfig, builder: EvalBuilder):
+
+    def load_excel(file_path, **kwargs):
+        return pd.read_excel(file_path, engine="openpyxl", **kwargs)
+
+    yield DatasetStoreInfo(config=config, load_fn=load_excel, description="Excel file dataset store")
+
+
+@register_dataset_store(config_type=EvalDatasetCustomConfig)
+async def register_custom_dataset_store(config: EvalDatasetCustomConfig, builder: EvalBuilder):
```

Ruff (0.14.14)

[warning] Unused function argument: `builder`

(ARG001)